.azure-pipelines.yml

```diff
@@ -11,35 +7,52 @@ trigger:
 pr:
 - master
 
-jobs:
-- template: /pipelines/corefxlab-base.yml
-  parameters:
-    name: Windows_NT
-    displayName: Innerloop Windows_NT
-    targetOS: Windows_NT
-
-    ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-      isOfficialBuild: true
-      pool:
-        name: NetCoreInternal-Pool
-        queue: buildpool.windows.10.amd64.vs2017
-    ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-      pool:
-        name: Hosted VS2017
+variables:
+  - name: _DotNetArtifactsCategory
+    value: EXPERIMENTAL
+  - name: _TeamName
+    value: DotNetCore
 
-- ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-  - template: /pipelines/corefxlab-base.yml
+stages:
+- stage: build
+  displayName: build
+  jobs:
+  - template: corefxlab-base.yml
     parameters:
-      name: MacOS
-      displayName: Innerloop OSX
-      targetOS: OSX
-      pool:
-        name: Hosted macOS
+      name: Windows_NT
+      displayName: Innerloop Windows_NT
+      targetOS: Windows_NT
+
+      ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        isOfficialBuild: true
+        pool:
+          name: NetCoreInternal-Pool
+          queue: buildpool.windows.10.amd64.vs2017
+      ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+        pool:
+          name: Hosted VS2017
+
+  - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+    - template: corefxlab-base.yml
+      parameters:
+        name: MacOS
+        displayName: Innerloop OSX
+        targetOS: OSX
+        pool:
+          name: Hosted macOS
+
+    - template: corefxlab-base.yml
+      parameters:
+        name: Ubuntu1604
+        displayName: Innerloop Ubuntu16.04
+        targetOS: Ubuntu
+        pool:
+          name: Hosted Ubuntu 1604
 
-  - template: /pipelines/corefxlab-base.yml
+  # Publish and validation steps. Only run in official builds
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: \eng\common\templates\post-build\post-build.yml
     parameters:
-      name: Ubuntu1604
-      displayName: Innerloop Ubuntu16.04
-      targetOS: Ubuntu
-      pool:
-        name: Hosted Ubuntu 1604
+      validateDependsOn:
+        - build
+      enableSymbolValidation: false # https://github.com/dotnet/arcade/issues/2871
```

Comment:
This issue has been closed for a few months. Do we still need this?

Reply:
Even though you are right(the issue has been closed for a while), Most repos still have this as I believe we haven't yet figured out a way to do symbol validation correctly. Here is a pointer to the Arcade repo which has today in master the exact same clause:

https://github.com/dotnet/arcade/blob/master/azure-pipelines.yml#L181-L183

Reply:
@JohnTortugo - should this URL be linked somewhere else? It doesn't seem right for us to add a new setting with a link to a closed issue.